cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -156,6 +156,10 @@ message CreateSeismicRequest {
     TextHeader text_header = 8;  // Optionally set a custom text header
     BinaryHeader binary_header = 9;  // Optionally set a custom binary header
     bool copy_metadata = 11;  // If true, copy the metadata from the source seismic store.
+    oneof psm_trace_group_cutout {
+        bool psm_empty = 14;  // If true, will create a prestack-migrated seismic with an empty trace group cutout
```

Comment:
Hmm, is this needed? This would imply that the full cutout is empty, so why not just use `empty` in the `volume` field? I guess there's the question of what to put here when that is empty. Probably just leave the entire field as `null`?

Reply:
Yeah true, two empties doesn't make a lot of sense, null is fine here I suppose

Reply:
And I guess `psm_trace_group_cutout = null` means that we include all the traces in each included group.